### PR TITLE
Header intro section

### PR DESCRIPTION
**Pull request by wassimchakib**
In this feature, I added:
 • Navigation bar that contains Portfolio, About and Contact links
 • Intro section which contains social media links and content related to the owner
 • Mobile friendly design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <header id="header">
+            <h3 id="logo" class="content-padding"><a href="#intro">My Logo</a></h3>
+            <nav id="navbar">
+                <ul>
+                    <li class="content-padding"><a href="#">Portfolio</a></li>
+                    <li class="content-padding"><a href="#">About</a></li>
+                    <li class="content-padding"><a href="#">Contact</a></li>
+                </ul>
+            </nav>
+            <a href="#" class="content-padding" id="menu"><img src="images/menu-icon.png" alt="menu"></a>
```
Comment (cynthiainga) [inline index.html]: - [ ] The hamburger button has a `nav` tag for semantics reason, please kindly add your hamburger button into the `nav` tag 😁

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        </header>
```
Comment (cynthiainga) [inline index.html]: - [ ] This container of the section is a `header` tag which means the `header` tag is outside the section tag. Cause we have two sections in this project, the toolbar in the `header` tag and the headline in their `section` tag 😁 🙃

File: index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a href="#">Let's connect</a>
+            <ul>
+                <li><a href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+                <li><a href="#"><img src="images/linkedin.svg" alt="linkedin"></a></li>
+                <li><a href="#"><img src="images/medium.svg" alt="medium"></a></li>
+                <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
+                <li><a href="#"><img src="images/angellist.svg" alt="angelist"></a></li>
+            </ul>
+        </div>
```
Comment (cynthiainga) [inline index.html]: - [ ] This container should have its `section` tag as I explain above 🙃

File: style.css [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+  font-family: 'Poppins', Helvetica, sans-serif;
+}
+
+#intro {
+  width: 100%;
+  height: 100vh;
+  background-image: url(images/header.png);
+  background-position: 100%;
+  background-size: cover;
+}
```
Comment (cynthiainga) [inline style.css]: - [ ] Please kindly make sure you use the correct background image for the mobile version, your design should sticks as much as possible to the Figma design 🙃 🙃 
- [ ] Also the image has a border-radius at the left bottom 😉 😉 

Your design

Screen Shot 2022-09-21 at 17 49 52

Figma design

Screen Shot 2022-09-21 at 17 50 26

File: style.css [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+  font-size: 48px;
+  font-weight: 700;
+  max-width: 400px;
+  align-self: flex-start;
+}
+
+.container p {
+  font-size: 20px;
+  font-weight: 400;
+}
```
Comment (cynthiainga) [inline style.css]: - [ ] As required in the project your design should stick as much as possible to the Figma design (e.g. font, colors,...) but your title and paragraph don’t have the same color and font as the title and paragraph in the Figma design, please kindly fix that 😉 

Your design

Screen Shot 2022-09-21 at 18 02 53

Figma design 

Screen Shot 2022-09-21 at 18 03 20

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @wassimchakib, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] Used correct Github flow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 5 above)
Comment (zieeco) [reply style.css]: - [x] This issue is well resolved :heavy_check_mark:

File: style.css (hunk 4 above)
Comment (zieeco) [reply style.css]: - [x] This issue is well resolved :heavy_check_mark:

File: index.html (hunk 3 above)
Comment (zieeco) [reply index.html]: - [x] This issue is well resolved :heavy_check_mark:

File: index.html (hunk 1 above)
Comment (zieeco) [reply index.html]: - [x] This issue is well resolved :heavy_check_mark:

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @wassimchakib, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Good commit messages 👍 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**